script.js

```diff
@@ -0,0 +1,53 @@
+const btn = document.getElementById('btn-color')
+const spanOnOff = document.getElementById('onOff')
+const spanLastTime = document.getElementById('lastTime')
+
+loadDOMDocument()
+
+btn.addEventListener('click', () => {
+   onOff()
+   getNormalizedTime()
+})
+
+function loadDOMDocument() {
+   document.addEventListener('DOMContentLoaded', () => {
+      if (localStorage.length) {
+         btn.innerText = localStorage.getItem('btn')
```

Comment:
Це можна винести з умови, і тут же задати значення по замовчуванню
```suggestion
         btn.innerText = localStorage.getItem('btn') || 'почати';
```

Reply:
Так можна зробити і в інших подібних випадках, і зробити код простіше і чистіше